### PR TITLE
chore: Document default strategy

### PR DESCRIPTION
**Pull request by andreas-unleash**
Documenting Default Strategy 
## About the changes

Closes # [1-1023](https://linear.app/unleash/issue/1-1023/document-strategy-improvements)

### CODE REVIEW

File: website/docs/reference/project-default-strategy.md [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+
+The project default strategy feature is generally available starting with **Unleash 5.2.0**.
+
+:::
+
+A project's environment's **default strategy** specifies **the activation strategy that will be used when you enable a feature in that environment, if the environment has no defined strategies)**. 
+
+## Limitations
+
+The default project strategy uses the [Gradual Rollout activation strategy](/reference/activation-strategies.md) as it's base.  You can customise to your needs by using constraints or segments (enterprise only)
```
Comment (thomasheartman) [inline website/docs/reference/project-default-strategy.md:15]: This header doesn't seem to apply to the text, so I think we can just remove it. (Plus some minor text suggestions and links)

```suggestion
All default project strategies use the [gradual rollout activation strategy](activation-strategies.md). By default, the rollout 100%. You can customize the strategies by changing the rollout percentage and adding [constraints](strategy-constraints.mdx) and [segments](segments.mdx) as you would for any other strategy.
```

File: website/docs/reference/project-default-strategy.md [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+
+## Setting up
```
Comment (thomasheartman) [inline website/docs/reference/project-default-strategy.md:17]:
```suggestion
## Configuration
```

File: website/docs/reference/project-default-strategy.md [hunk 3]
```diff
@@ -0,0 +1,29 @@
...
+
+When in project overview, go to the Project settings tab and click on Default strategy
```
Comment (thomasheartman) [inline website/docs/reference/project-default-strategy.md:19]:
```suggestion
Custom strategies are configured from each project's project settings tab.
```

File: website/docs/reference/project-default-strategy.md [hunk 4]
```diff
@@ -0,0 +1,29 @@
+---
+title: Project default strategy
+---
+
```
Comment (thomasheartman) [inline website/docs/reference/project-default-strategy.md:4]: I suggest using figures for some of the images. To do that, we need to import the component.

```suggestion
import Figure from '@site/src/components/Figure/Figure.tsx'

```

File: website/docs/reference/project-default-strategy.md [hunk 5]
```diff
@@ -0,0 +1,29 @@
...
+## Limitations
+
+The default project strategy uses the [Gradual Rollout activation strategy](/reference/activation-strategies.md) as it's base.  You can customise to your needs by using constraints or segments (enterprise only)
+
+## Setting up
+
+When in project overview, go to the Project settings tab and click on Default strategy 
+
+![project_settings_default_strategy.png](/img/project_settings_default_strategy.png)
```
Comment (thomasheartman) [inline website/docs/reference/project-default-strategy.md:21]:
```suggestion
<Figure caption="The default strategy configuration page is available from the project settings tab." img="/img/project_settings_default_strategy.png" />
```

File: website/docs/reference/project-default-strategy.md [hunk 6]
```diff
@@ -0,0 +1,29 @@
...
+
+You can customise the default strategy for each of the project's environments by clicking on the edit button
+
+![edit_default_Strategy.png](/img/edit_default_Strategy.png)
```
Comment (thomasheartman) [inline website/docs/reference/project-default-strategy.md:25]:
```suggestion
The default strategies screen lists a strategy for each of the project's environments

<Figure caption="Each strategy can be individually configured with the corresponding edit button." img="/img/edit_default_Strategy.png" />
```

File: website/docs/reference/project-default-strategy.md [hunk 7]
```diff
@@ -0,0 +1,29 @@
...
+
+This will open up the familiar activation strategy form, where you can add your constraints and/or segments
+
+![activation_Strategy_form.png](/img/activation_Strategy_form.png)
```
Comment (thomasheartman) [inline website/docs/reference/project-default-strategy.md:29]: I don't think we need this, but am happy to hear your thoughts otherwise.
```suggestion
```

**Review by thomasheartman — CHANGES_REQUESTED** [review]
Nice job; I think this is looking very good 😄  I'd suggest moving the default strategy doc into the Projects document instead of creating a new doc, though. It feels like that would be the appropriate place. What do you think? if not, we should also make a new entry in the sidebar for it.

Also, regarding images: please rename the images to use hyphens instead of underscores and avoid capital letters -- just to keep the format the same with all the other images.

Otherwise, there's a few small suggestions, but most of it is all good.

File: website/docs/reference/project-default-strategy.md
[the remark this answers is not in the document]
Comment (andreas-unleash) [reply website/docs/reference/project-default-strategy.md:1]: sure makes sense

File: website/docs/reference/project-default-strategy.md (hunk 7 above)
Comment (andreas-unleash) [reply website/docs/reference/project-default-strategy.md:30]: makes sense

File: website/docs/reference/projects.md [hunk 8]
```diff
@@ -98,3 +98,28 @@ All available projects are available from the drop-down menu.
 If you want to change which project a feature toggle belongs to, you can change that from the feature toggle's configuration page. Under the _settings_ tab, choose the _project_ option and choose the new project from the dropdown menu.
 
 ![A feature toggle's settings tab. The project setting shows a dropdown to change projects.](/img/projects_existing_toggle_dropdown.png)
+
+---
+title: Project default strategy
+---
```
Comment (thomasheartman) [inline website/docs/reference/projects.md:104]:
```suggestion
## Project default strategy
```

File: website/docs/reference/projects.md [hunk 9]
```diff
@@ -98,3 +98,28 @@ All available projects are available from the drop-down menu.
...
+
+The project default strategy feature is generally available starting with **Unleash 5.2.0**.
+
+:::
+
+You can define default strategies for each of a project's environments. The default strategy for an environment will be added to a feature when you enable it in an environment **if and only if** the feature has **no active strategies** defined.
+
+All default project strategies use the [gradual rollout activation strategy](activation-strategies.md). By default, the rollout 100%. You can customize the strategies by changing the rollout percentage and adding [constraints](strategy-constraints.mdx) and [segments](segments.mdx) as you would for any other strategy.
+
+## Configuration
```
Comment (thomasheartman) [inline website/docs/reference/projects.md:117]:
```suggestion
### Configuration
```

**Review by thomasheartman — APPROVED** [review]
Looks good! Just two more suggestions to adjust the heading levels based on the position, but otherwise all good 👏🏼 

That should also fix the deployment issues.

File: website/docs/reference/projects.md [hunk 10]
```diff
@@ -98,3 +98,26 @@ All available projects are available from the drop-down menu.
...
+
+:::info Availability
+
+The project default strategy feature is generally available starting with **Unleash 5.2.0**.
+
+:::
+
+You can define default strategies for each of a project's environments. The default strategy for an environment will be added to a feature when you enable it in an environment **if and only if** the feature has **no active strategies** defined.
+
+All default project strategies use the [gradual rollout activation strategy](activation-strategies.md). By default, the rollout 100%. You can customize the strategies by changing the rollout percentage and adding [constraints](strategy-constraints.mdx) and [segments](segments.mdx) as you would for any other strategy.
```
Comment (thomasheartman) [inline website/docs/reference/projects.md:113]: Sorry, my bad: the link here was wrong 😳 

```suggestion
All default project strategies use the [gradual rollout activation strategy](activation-strategies.md). By default, the rollout 100%. You can customize the strategies by changing the rollout percentage and adding [constraints](strategy-constraints.mdx) and [segments](segments.md) as you would for any other strategy.
```

File: website/docs/reference/projects.md [hunk 11]
```diff
@@ -98,3 +98,26 @@ All available projects are available from the drop-down menu.
...
+
+:::info Availability
+
+The project default strategy feature is generally available starting with **Unleash 5.2.0**.
+
+:::
+
+You can define default strategies for each of a project's environments. The default strategy for an environment will be added to a feature when you enable it in an environment **if and only if** the feature has **no active strategies** defined.
+
+All default project strategies use the [gradual rollout activation strategy](activation-strategies.md). By default, the rollout 100%. You can customize the strategies by changing the rollout percentage and adding [constraints](strategy-constraints.mdx) and [segments](segments.md) as you would for any other strategy.
```
Comment (thomasheartman) [inline website/docs/reference/projects.md:113]: Nope, the other way 🙈 

```suggestion
All default project strategies use the [gradual rollout activation strategy](activation-strategies.md). By default, the rollout 100%. You can customize the strategies by changing the rollout percentage and adding [constraints](strategy-constraints.md) and [segments](segments.mdx) as you would for any other strategy.
```

**Pull request merged**